unity-renderer/Assets/DCLServices/Lambdas/LambdasService.cs

```diff
@@ -74,6 +74,7 @@ internal string GetUrl(string endPoint, params (string paramName, string paramVa
             var urlBuilder = GenericPool<StringBuilder>.Get();
             urlBuilder.Clear();
             urlBuilder.Append(GetLambdasUrl());
+            urlBuilder.Append('/');
```

Comment:
Please be defensive with this one, since many configurations may occur in different realms/worlds

```suggestion
            if(!urlBuilder.endsWith('/')) urlBuilder.Append('/');
```

Reply:
I have checked that we could put always this final `/` as it would be the same to request using this url:
`https://peer.decentraland.org/lambdas/users/...`
or this one:
`https://peer.decentraland.org/lambdas/users//...`

Anyway, if you are not convinced, I can do the check in the code.

Reply:
Done! https://github.com/decentraland/unity-renderer/pull/4684/commits/36ceb55a063072ddb02c6d4893cc68ab42639b4e